answerking_app/utils/mixins/CategoryItemMixins.py

```diff
@@ -20,16 +18,7 @@ def update(
         item: Item = get_object_or_404(Item, pk=item_id)
 
         if item in category.items.all():
-            error_msg: ErrorMessage = {
-                "error": {
-                    "message": "Resource update failure",
-                    "details": "Item already in category",
-                }
-            }
-            return Response(
-                error_msg,
-                status=status.HTTP_400_BAD_REQUEST,
-            )
+            raise HttpErrorResponse(status=status.HTTP_400_BAD_REQUEST)
```

Comment:
How is the detail from the error captured?

Reply:
DRF handles the exceptions, I defined the content in `HttpErrorResponse`